src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj

```diff
@@ -12,12 +12,13 @@
   </PropertyGroup>
     <ItemGroup>
         <PackageReference Include="CommandLineParser" Version="2.8.0" />
+        <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2767" />
+        <PackageReference Include="Greg" Version="3.0.0.2926" />
         <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" />
```

Comment:
Same here, still seeing 2 versions of libg nuget.

Reply:
thanks @aparajit-pratap - totally missed that as I did not touch LibG in the nuget UI so did not expect it to be affected, I will fix it.

Reply:
I think you might need to pull the latest master to your PR branch if you haven't already done so before making these commits.

Reply:
thanks @aparajit-pratap I need to update to whatever the actual published version of Greg.dll ends up being so I will do that then.